Applications/Spire/Include/Spire/BookView/BookViewHighlightPropertiesWidget.hpp

```diff
@@ -3,6 +3,7 @@
 #include <QListWidget>
 #include <QWidget>
 #include "Spire/BookView/BookView.hpp"
+#include "Spire/Spire/Spire.hpp"
```

Comment:
Only include Signal.hpp unless you otherwise need Spire.hpp. Applies to a few more occurrences below.